src/applications/disability-benefits/all-claims/pages/uploadForm4192.jsx

```diff
@@ -0,0 +1,62 @@
+import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import environment from '../../../../platform/utilities/environment';
+
+import { DocumentDescription } from '../content/uploadPtsdDocuments';
+
+const FIFTY_MB = 52428800;
+
+export const uiSchema = {
+  'ui:title': 'Upload VA Form 21-4192',
+  'ui:description': DocumentDescription,
+  uploaded4192: fileUploadUI('', {
+    itemDescription: '21-4192 form',
```

Comment:
If this is the only thing that's changing between all these file upload fields, would it make sense to create another little helper function like:
```js
function ancillaryFormUploadUI(itemDescription) {
  return fileUploadUI('', {
    itemDescription,
    ...
  }
}
```

Helper functions for helper functions, I know, but if it helps keeps things consistent, it might be an easy win.